client/ayon_houdini/plugins/publish/extract_rop.py

```diff
@@ -19,6 +19,7 @@ def process(self, instance: pyblish.api.Instance):
         if instance.data.get("farm"):
             self.log.debug("Should be processed on farm, skipping.")
             return
+        creator_attribute = instance.data["creator_attributes"]
```

Comment:
Do be wary that _technically_ an instance may not have `creator_attributes` if it's e.g. a runtime instance and not all data is transferred from a CreateContext instance. It may be worth (for how generic this extractor is) to do:

```suggestion
        creator_attribute = instance.data.get("creator_attributes", {})
```